src/prefect/tasks/aws/waiters/batch.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Assuming you all want to keep these custom waiter definitions in, they should probably be documented somewhere, since they are not part of the standard boto3 waiter library. I wasn't sure where would be most appropriate.

Reply:
@madelgi I think the best place for them would be in the function docstrings for the tasks that is viewable from the API Reference documentation since that is the natural place for this kind of task-specific information